Dockerfile

```diff
@@ -1,5 +1,17 @@
+FROM golang:alpine as builder
+
+WORKDIR /go/src/github.com/blendle/pg2kafka
+ADD . ./
```

Comment:
I would not use `ADD` at this line but later, after adding librdkafka to reuse some cache